installers/olm/postgresoperator.csv.images.yaml

```diff
@@ -13,15 +13,15 @@
 - { name: RELATED_IMAGE_PGO_RMDATA,             value: '${PGO_IMAGE_PREFIX}/pgo-rmdata:${PGO_IMAGE_TAG}' }
 - { name: RELATED_IMAGE_PGO_SQL_RUNNER,         value: '${PGO_IMAGE_PREFIX}/pgo-sqlrunner:${PGO_IMAGE_TAG}' }
 
-- { name: RELATED_IMAGE_CRUNCHY_ADMIN,            value: '${CCP_IMAGE_PREFIX}/crunchy-admin:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_BACKREST_RESTORE, value: '${CCP_IMAGE_PREFIX}/crunchy-backrest-restore:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_COLLECT,          value: '${CCP_IMAGE_PREFIX}/crunchy-collect:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_GRAFANA,          value: '${CCP_IMAGE_PREFIX}/crunchy-grafana:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PGADMIN,          value: '${CCP_IMAGE_PREFIX}/crunchy-pgadmin4:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PGBADGER,         value: '${CCP_IMAGE_PREFIX}/crunchy-pgbadger:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PGBOUNCER,        value: '${CCP_IMAGE_PREFIX}/crunchy-pgbouncer:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PGDUMP,           value: '${CCP_IMAGE_PREFIX}/crunchy-pgdump:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PGRESTORE,        value: '${CCP_IMAGE_PREFIX}/crunchy-pgrestore:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_POSTGRES_HA,      value: '${CCP_IMAGE_PREFIX}/crunchy-postgres-ha:${CCP_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_POSTGRES_GIS_HA,  value: '${CCP_IMAGE_PREFIX}/crunchy-postgres-gis-ha:${CCP_POSTGIS_IMAGE_TAG}' }
-- { name: RELATED_IMAGE_CRUNCHY_PROMETHEUS,       value: '${CCP_IMAGE_PREFIX}/crunchy-prometheus:${CCP_IMAGE_TAG}' }
+- { name: RELATED_IMAGE_CRUNCHY_ADMIN,               value: '${CCP_IMAGE_PREFIX}/crunchy-admin:${CCP_IMAGE_TAG}' }
+- { name: RELATED_IMAGE_CRUNCHY_BACKREST_RESTORE,    value: '${CCP_IMAGE_PREFIX}/crunchy-backrest-restore:${CCP_IMAGE_TAG}' }
+- { name: RELATED_IMAGE_CRUNCHY_POSTGRES_EXPORTER,   value: '${CCP_IMAGE_PREFIX}/crunchy-postgres-exporter:${CCP_IMAGE_TAG}' }
```

Comment:
Images built in this project are tagged using `PGO_IMAGE_PREFIX`.

Reply:
Updated, thanks.